### PR TITLE
Updated UI of product page #7522

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

**Comment by gautampanday** [comment]
faded short sleeves t shirt nereid webshop

File: templates/webshop/product.jinja [hunk 1]
```diff
@@ -79,27 +79,20 @@
...
 
       {% block product_name_price %}
       <h3 class="product-heading product-name">
-        <span itemprop="name">{{ product.name }}</span>
+        <span itemprop="name" class="heading-500">{{ product.name }}</span>
       </h3>
+      {% if product.code %}
       <h4>
         <small class="product-code" itemprop="identifier" content="sku:{{ product.code }}">{{ product.code }}</small>
+
```
Comment (tarunbhardwaj) [inline templates/webshop/product.jinja]: remove

File: templates/webshop/product.jinja [hunk 2]
```diff
@@ -123,7 +116,23 @@
...
+          {% if sale_price < product.list_price %}
+          <h4 class="product-price heading-500">
+          <span class="not">{{ product.list_price|currencyformat(request.nereid_currency.code) }}</span>
+          <span itemprop="price">{{ sale_price|currencyformat(request.nereid_currency.code) }}</span>
+        </h4>
+        {% else %}
+        <h4 class="product-price product-list-price" itemprop="price">{{ sale_price|currencyformat(request.nereid_currency.code) }}</h4>
+        {% endif %}
+      </div>
+      {% endblock product_price %}
```
Comment (tarunbhardwaj) [inline templates/webshop/product.jinja]: fix indentation

File: templates/webshop/product.jinja [hunk 3]
```diff
@@ -123,7 +116,23 @@
       <span class="text-warning" style="display:none" id="product-not-available">
         <i class="fa fa-warning"></i> {{ _('This product is not available') }}
       </span>
-
+    </div>
+    <div class="col-md-3 col-sm-3 col-xs-12">
+    <div class="buy-box">
+    {% block product_price %}
```
Comment (tarunbhardwaj) [inline templates/webshop/product.jinja]: fix indentation in this block

Comment (tarunbhardwaj) [inline templates/webshop/product.jinja]: no need of block

**Pull request merged**